src/adapters/org.mozilla-iot.adapters.example-plugin/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "api": 1,
+    "id": "org.mozilla-iot.adapters.example-plugin",
+    "name": "Example Plugin",
+    "plugin": true,
+    "config": {
+    }
```

Comment:
Presumably we can remove the config from the manifest for adapters which don't use it.